core/arch/arm/plat-nuvoton/conf.mk

```diff
@@ -0,0 +1,49 @@
+PLATFORM_FLAVOR ?= npcm845x
+
+ifeq ($(PLATFORM_FLAVOR),npcm845x)
+include core/arch/arm/cpu/cortex-armv8-0.mk
+CFG_ARM64_core ?= y
+endif #npcm845x
+CFG_USER_TA_TARGETS ?= ta_arm64
+DEBUG ?= 1
+
+ifeq ($(platform-debugger-arm),1)
+# ARM debugger needs this
+platform-cflags-debug-info = -gdwarf-2
+platform-aflags-debug-info = -gdwarf-2
+endif
+
+ifeq ($(platform-flavor-armv8),1)
+CFG_WITH_ARM_TRUSTED_FW ?= y
+endif
+
+$(call force,CFG_GIC,y)
+$(call force,CFG_ARM_GICV2,y)
+$(call force,CFG_SECURE_TIME_SOURCE_CNTPCT,y)
+$(call force,CFG_16550_UART,y)
+
+ifeq ($(CFG_CORE_TPM_EVENT_LOG),y)
+# NOTE: Below values for the TPM event log are implementation
+# dependent and used mostly for debugging purposes.
+# Care must be taken to properly configure them if used.
+# CFG_TPM_LOG_BASE_ADDR ?= 0x402c951
+# CFG_TPM_MAX_LOG_SIZE ?= 0x200
+endif
```

Comment:
prefer you remove this.
You'll be able to restore tpm log support when the platform is ready for it.

Reply:
Done